cmake/DetectOptions.cmake

```diff
@@ -395,7 +395,9 @@ if(NOT SHARED_LIBS_SUPPORTED)
   endif()
 endif()
 
-if(ADIOS2_USE_Python STREQUAL AUTO)
+if(ADIOS2_PIP_INSTALL)
```

Comment:
this option could be called ADIOS2_USE_PIP to follow the same standard as other adios2 options

Reply:
I Would also added it as an adios2 standard option

Reply:
I'll change the name, but what do you mean add it as a "standard option"?

Reply:
Something like this https://github.com/ornladios/ADIOS2/blob/master/CMakeLists.txt#L174

Also here https://github.com/ornladios/ADIOS2/blob/master/CMakeLists.txt#L244

Reply:
I see.  My opinion was it should be marked as advanced, and not exposed as an option to users, as it shouldn't be needed or used outside of building with pip.

Reply:
we can declare the options as advanced. The reason of this is to have it printout when we do configure, this might come handy in future troubleshooting sessions

Reply:
It seems that when we do the printout in configure, it's checking for `ADIOS2_HAVE_OPTION`, but the `adios_option` function only defines `ADIOS2_USE_OPTION`.  Are we supposed to define manually the "`_HAVE_`" variable in `DetectOptions.cmake`?

Reply:
> Are we supposed to define manually the "_HAVE_" variable in DetectOptions.cmake?

Yes, thats were we add the logic for it.